pulpcore/pulpcore/app/models/publication.py

```diff
@@ -14,6 +15,7 @@ class Publication(Model):
             create this Publication.
     """
 
+    complete = models.BooleanField(default=False)
```

Comment:
The complete field needs an index.

Reply:
I guess that's to make the `.exclude()` effective? Will update..